Fixes ltmain.sh to properly add SYSROOT prefix to library
search pathes it's adding silently.

Enrico Weigelt <weigelt@metux.de>

diff --ruN a/ltmain.sh b/ltmain.sh
--- a/ltmain.sh	2010-04-29 10:10:16.000000000 +0000
+++ b/ltmain.sh	2010-08-18 11:23:48.000000000 +0000
@@ -5768,7 +5768,7 @@
 	       test "$hardcode_direct_absolute" = no; then
 	      add="$libdir/$linklib"
 	    elif test "$hardcode_minus_L" = yes; then
-	      add_dir="-L$libdir"
+	      add_dir="-L${SYSROOT}$libdir"
 	      add="-l$name"
 	    elif test "$hardcode_shlibpath_var" = yes; then
 	      case :$finalize_shlibpath: in
@@ -5785,7 +5785,7 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir="-L$libdir"
+	      add_dir="-L${SYSROOT}$libdir"
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
